src/main/java/de/qabel/desktop/ui/accounting/AccountingController.java

```diff
@@ -94,7 +96,7 @@ protected void addIdentityWithAlias(String alias) {
 	protected void handleImportIdentityButtonAction(ActionEvent event) {
 
 		FileChooser chooser = new FileChooser();
-		chooser.setTitle("Choose Download Folder");
+		chooser.setTitle(resourceBundel.getString("downloadFolder"));
```

Comment:
we should talk about these identifiers. Probably some conventions would make sense. Resulting in identitifers like
`DEFAULT_IDENTITY_ALIAS`
`DOWNLOAD_DIALOG_TITLE`
`NAVI_TAB_IDENTITIES`
which are hopefully understandable without their default translation